src/Plugin.Maui.Audio/AudioRecorder/IAudioRecorder.shared.cs

```diff
@@ -26,6 +26,20 @@ public interface IAudioRecorder
 	///<param name="filePath">The path on disk to store the recording.</param>
 	Task StartAsync(string filePath);
 
+	///<Summary>
+	/// Start recording audio to disk in a randomly generated file. AudioRecordingOptions are only supported on Android and iOS.
```

Comment:
Just out of curiosity, is it not possible on Windows or we just don't know how?

Reply:
I just don't know. But I can look into it!